portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,15 +23,36 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that sends and receives comments data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private ArrayList<String> data = new ArrayList(Arrays.asList("Message 0", "This is message 1.", "MESSAGE TWO"));
+  private ArrayList<String> comments = new ArrayList();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Gson gson = new Gson();
     response.setContentType("text/html;");
-    response.getWriter().println(gson.toJson(data));
+    response.getWriter().println(gson.toJson(comments));
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Store submitted comment
+    comments.add(getParameter(request, "comment-submission", ""));
```

Comment:
What does the UI look like if we end up with several comments of empty strings?

Reply:
I was planning on fixing the UI for displaying comments in a separate PR since this one is already large. The UI currently looks like this:
![image](https://user-images.githubusercontent.com/66034575/84038187-9c7a1f80-a96d-11ea-80cb-acef3eb5ce59.png)